cogs/playerstats.py

```diff
@@ -141,6 +154,48 @@ def getAssists(comp):
     table = 'Assist Stats: ' + comp + '\n' + table
     return table
 
+def getInjuries(team="Arsenal"):
+    tableurl = "https://www.premierinjuries.com/injury-table.php"
+    #get table and convert to dataframe
+    header = {
+        "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.75 Safari/537.36",
+        "X-Requested-With": "XMLHttpRequest"
+    }
+    playerDict = {"Player": "", "Reason": "", "FurtherDetail": "", "PotentialReturn": "", "Condition": "", "Status": ""}
+    injuries = []
+    r = requests.get(tableurl).text
+    bs_obj = BeautifulSoup(r, features="lxml")
+    #currently team_1 is arsenal, but ideally i'd like to have some sort of error-checky way
+    # to make sure that it looks for arsenal and confirms that the team ID is 1.  
+    #i'll let future ren figure that out
+    fullTable = bs_obj.find_all('tr', attrs={'class': "player-row team_1"})
+    playerList = [i.text for i in fullTable]
+    for plObj in playerList:
+        playerDict = {"Player": "", "Reason": "", "FurtherDetail": "", "PotentialReturn": "", "Condition": "", "Status": ""}
+        plObj = plObj.split('\n')
+        plObj.pop(0)
+        player = plObj.pop(0)
+        player = player.rsplit("Player")[-1]
```

Comment:
Personally not a fan of defining a var then immediately just assigning that var to something else without transforming it again. Would prefer to see as below but that's purely opinion

```
player = plObj.pop(0)
playerDict["Player"] = player.rsplit("Player")[-1]
```

Reply:
Well I have to pop it twice before it gets to the first player.  So would you rather I go...

```
player = plObj.split('\n')
player.pop(0)
player.pop(0)
player = player.rsplit("Player")[-1]
```
?